lib/repository/repository.ts

```diff
@@ -14,6 +15,7 @@ type Repository<T extends IdEntity> = {
 	find: <S extends keyof T>(options?: FindOptions<T, S>) => Promise<Pick<T, S>[]>;
 	findOne: <S extends keyof T>(options: T['id'] | FindOneOptions<T, S>) => Promise<Pick<T, S> | null>;
 	insert: (data: InsertData<T>) => Promise<{ id: T['id'] }>;
+	insertMany: (data: InsertData<T>[]) => Promise<{ id: T['id'] }[]>;
```

Comment:
Technically it should be multi right? Too keep it consistent?

Reply:
No, I had to chose multi because of naming collisions, I would have loved to call the db function insertMany